libbpf-tools/checker_helpers.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef CHECKER_HELPERS_H
+#define CHECKER_HELPERS_H
+
+#include <unistd.h>
+#include <stdlib.h>
+
+static inline int probe_fd(int fd) {
```

Comment:
It would be better to remove this function and add the logic directly in .c file.

Reply:
removed.

Reply:
`#include <stdlib.h>` should be removed too.

Reply:
removed.